.github/workflows/nightly-tests.yml

```diff
@@ -58,6 +58,18 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+
+      - name: Get the helm chart versions for the perf env
+        run: |
+          curl -H 'Authorization: token ${{ secrets.BROADBOT_TOKEN }}' \
+            -H 'Accept: application/vnd.github.v3.raw' \
+            -L https://api.github.com/repos/broadinstitute/terra-helmfile/contents/versions/app/dev.yaml \
+            --create-dirs -o "integration/src/main/resources/rendered/dev.yaml"
+          curl -H 'Authorization: token ${{ secrets.BROADBOT_TOKEN }}' \
+            -H 'Accept: application/vnd.github.v3.raw' \
+            -L https://api.github.com/repos/broadinstitute/terra-helmfile/contents/environments/live/perf.yaml \
+            --create-dirs -o "integration/src/main/resources/rendered/perf.yaml"
```

Comment:
There might be a difference between what is in the helm charts and what is deployed to the dev / perf env. Should we curl the dev / perf version endpoints instead?

Reply:
I totally agree! However the current QA dashboard requires a helm chart version which our version endpoint doesn't return. I've talked with the QA team and the plan is that the next version of testrunner will support using the `version` endpoint of a service to get this information.

Reply:
It would be much better to get the version information from this endpoint, and would allow us to remove this `curl` code. But for now this is the "best" way to do this.